source/administration/upgrade.rst

```diff
@@ -0,0 +1,232 @@
+Upgrading Mattermost Server
+===========================
+
+In most cases, you can upgrade Mattermost Server in a few minutes, depending on how long it takes for you to back up both the database and the file storage directory.
```

Comment:
`in most cases` unnecessary? Implied by the latter part of the sentence.

I'm also not sure this is true as it took Shift almost an hour to upgrade to 3.8 due to a DB change to the posts table.